DataRecord/examples/0.2-DRAFT_examples/MobiDB_jsonld.json

```diff
@@ -0,0 +1,46 @@
+{
+  "@context": "http://schema.org",
+  "@type": "DataRecord",
+  "@id": "https://mobidb.org/P0DTC9#DR",
+  "dct:conformsTo": "https://bioschemas.org/profiles/DataRecord/0.2-DRAFT-2019_06_14",
+  "includedInDataset": "https://mobidb.org/#2020-03",
+  "mainEntity": {
+    "@type": "Protein",
+    "@id": "https://mobidb.org/P0DTC9",
+    "dct:conformsTo": "https://bioschemas.org/profiles/Protein/0.11-RELEASE",
+    "identifier": "https://identifiers.org/mobidb:P0DTC9",
```

Comment:
If people will see only P0DCT9 as identifier on the web page, I suggest using only that. @AlasdairGray what do you think?